### DISCUSSION TITLE
Authentication Issue: Slack auth redirecting back to login page after authorizing app

### DISCUSSION THREAD

**Discussion by okontsograda** [outline/outline#3788] [Self Hosting] [answered]
I have setup Outline with the latest stable build and I'm using it with the slack auth. I'm able to get everything working to get to the login screen, except that when I authenticate, the slack application it redirects back to the login screen and I see the following in the log from outline:

`{"error":"The maximum number of teams has been reached","stack":"UnauthorizedError: The maximum number of teams has been reached\n    at AuthenticationError (/home/ubuntu/outline/build/server/errors.js:35:34)\n    at accountProvisioner (/home/ubuntu/outline/build/server/commands/accountProvisioner.js:45:43)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Strategy._verify (/home/ubuntu/outline/build/server/routes/auth/providers/slack.js:56:22)","level":"error","message":"Error during authentication"}`

Edit: 

I made sure that I'm only setting a single authentication method and since the installation was clean I don't see how it's possible that I could be reaching more teams than 1.

The furthest I'm able to get is the Slack Authorication screen:
![image](https://user-images.githubusercontent.com/22104497/179064351-bdfe4ffa-1b3d-46a0-a690-ad1f23b8f006.png)

I've triple checked that I've setup everything correctly in the .env file and I've dropped the db, re-created and even re-installed the entire application to no avail. Any help would be apprecaited.

**Comment by tadopionero** [comment]
same issue

**Comment by Pashaflayt** [comment]
This was fixed in 0.65.2, make sure you have the latest release.

**Reply by tadopionero** [reply]
still had this bug even my version in 0.65.2
```
outline_1   | {"error":"The maximum number of teams has been reached","stack":"UnauthorizedError: The maximum number of teams has been reached\n    at AuthenticationError (/opt/outline/build/server/errors.js:35:34)\n    at accountProvisioner (/opt/outline/build/server/commands/accountProvisioner.js:45:43)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Strategy._verify (/opt/outline/build/server/routes/auth/providers/slack.js:56:22)","level":"error","message":"Error during authentication"}
nginx_1     | 118.70.32.198 - - [03/Aug/2022:11:19:52 +0000] "GET /auth/slack.callback?code=3464225924784.3889307358626.9135c8543a066c4d43b9d0a8e19fed931f8054c762675ffba77c17bfe04fd44d&state=f68b5f65cdb54129 HTTP/1.1" 302 145 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36" "-"
```

**Comment by okontsograda** [comment, marked as answer]
I was able to resolve the issue by making sure I checked out a specific tag with git. I had checked out the most recent version, but never updated git to checkout the tag.

If are experiencing something similar, this is what I did to fix the issue:

Fetch all of the tags available
`git fetch --all --tags`

Checkout the most recent reliable tag(version)
`git checkout tags/<tag> -b <branch>
`

Hope this helps someone!
